tools/stimulus/types-RR.yaml

```diff
@@ -546,3 +546,8 @@ VERTEX_WEIGHTS:
         } else {
           %I% <- NULL
         }
+
+MSTALGORITHM:
+    DEFAULT:
+        AUTOMATIC: c("automatic", "unweighted", "prim", "kruskal")
```

Comment:
This is technically sufficient, but I strongly recommend setting up all values (PRIM, KRUSKAL, etc.), otherwise someone will get confused in the future when just for this one type (MSTALGORITHM) only some values work, while for other types all values work.